ph5/utilities/resp_load.py

```diff
@@ -138,11 +144,6 @@ def create_list(self):
                             bit_weight = Response_t['bit_weight/value_d']
                             bit_weight_units = Response_t['bit_weight/units_s']
                             gain_units = Response_t['gain/units_s']
-                        else:
-                            LOGGER.warning(
-                                "No Response table found for das {0} channel "
-                                "{1}.\n".format(serial, channel))
-                        try:
```

Comment:
Why remove the warning?

Reply:
If you run the program on a master.ph5 file that hasn't had data loaded yet you'll potentially receiver thousands of lines of warnings. IMO, this just becomes noise.

Is there a reason that data needs to be loaded before a response is added to the master.ph5 file? Why should it make a difference if someone loads metadata into the master.ph5 before loading the timeseries?